exec/java-exec/src/main/java/org/apache/drill/exec/proto/helper/QueryIdHelper.java

```diff
@@ -33,7 +33,7 @@ public static String getQueryId(final QueryId queryId) {
 
```

Comment:
Please update the method `getQueryId` at [line 30](https://github.com/apache/drill/pull/1301/files#diff-8fabc8951e7dd22401fb82ee09e4ea6dR30) as below:
```
public static String getQueryId(final QueryId queryId)  {
    return queryId.hasText() ? queryId.getText() : (new UUID(queryId.getPart1(), queryId.getPart2())).toString();
}
```
Revert the change from here:

https://github.com/apache/drill/blob/master/exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileWrapper.java#L66